src/controller_examples/rabbitmq_controller/spec/resource/stateful_set.rs

```diff
@@ -75,12 +75,6 @@ impl ResourceBuilder<RabbitmqClusterView, RabbitmqReconcileState> for StatefulSe
         let made_spec = make_stateful_set(rabbitmq, new_strlit("")@).spec.get_Some_0();
         &&& StatefulSetView::unmarshal(object).is_Ok()
         &&& sts.spec.is_Some()
-        &&& made_spec == StatefulSetSpecView {
```

Comment:
Why is this deleted? I thought you needed it to say the fields other than replicas, templated and retention policy remains the same as the one in make_stateful_set

Reply:
One last question on this: seems to me you can even make this spec fn return `true`, like the one for service object?

Reply:
Why? I think leaving it here is fine